Update do_install

OliverKohl opened the pull request.

Commit:
Update do_install

imageguy commented:
Hi, I see you added a 4.8 patch and also switched to 14.502 installer. What Fedora are you running on?  4.7 patch and 15.302 installer run fine on my 4.8.10-200.fc24.x86_64. If we update do_install, we'd need to provide the 4.8 patch (I don't have one in my master branch) and put in the right selection logic to cover the right kernels. Thanks, Nenad

OliverKohl commented:
Dear Nenad,
I've forked your repository, the working 4.8 patch can be found there. Currently I don't use fedora, I use debian, I'm on testing (unstable) branch, your code works totally fine, but I needed to upgrade that's why I need 4.8.
Thanks for sharing your work and your ideas on GitHub.
Best regards
Oliver
-------- Original message --------From: imageguy <notifications@github.com> Date: 1/6/17  18:04  (GMT+01:00) To: imageguy/fglrx-for-Fedora <fglrx-for-Fedora@noreply.github.com> Cc: "Oliver Kohl D.Sc." <oliver@kohl.bz>, State change <state_change@noreply.github.com> Subject: Re: [imageguy/fglrx-for-Fedora] Update do_install (#9) 
Hi, I see you added a 4.8 patch and also switched to 14.502 installer. What Fedora are you running on?  4.7 patch and 15.302 installer run fine on my 4.8.10-200.fc24.x86_64. If we update do_install, we'd need to provide the 4.8 patch (I don't have one in my master branch) and put in the right selection logic to cover the right kernels. Thanks, Nenad

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/imageguy/fglrx-for-Fedora","title":"imageguy/fglrx-for-Fedora","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/imageguy/fglrx-for-Fedora"}},"updates":{"snippets":[{"icon":"PERSON","message":"@imageguy in #9: Hi, I see you added a 4.8 patch and also switched to 14.502 installer. What Fedora are you running on?  4.7 patch and 15.302 installer run fine on my 4.8.10-200.fc24.x86_64. If we update do_install, we'd need to provide the 4.8 patch (I don't have one in my master branch) and put in the right selection logic to cover the right kernels. Thanks, Nenad"}],"action":{"name":"View Pull Request","url":"https://github.com/imageguy/fglrx-for-Fedora/pull/9#issuecomment-270950902"}}}

Commit:
Merge branch 'master' into patch-1
